Tasks/UseDotNetV2/task.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "UseDotNet",
+    "friendlyName": "Use .NET Core",
+    "description": "Acquires a specific version of the .NET Core SDK from the internet or the local cache and adds it to the PATH. Additionally provides proxy support.",
+    "category": "Tool",
+    "preview": true,
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "DotNetCore"
+    ],
+    "demands": [],
+    "ecosystem": ".NET Core",
+    "instanceNameFormat": "Use .NET Core",
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "required": false,
+            "helpMarkDown": "Version to use.  Examples: 1.1.13, 2.2.102"
+        },
+        {
+            "name": "packageType",
```

Comment:
@vtbassmatt consider: should we just yank this option for the package type?

Reply:
The `use` usage won't need it. I'm fine leaving it as an option for the full task.